services/hanke-service/src/main/kotlin/fi/hel/haitaton/controllers/HankeController.kt

```diff
@@ -0,0 +1,32 @@
+package fi.hel.haitaton.controllers
+
+import fi.hel.haitaton.data.Hanke
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+import java.util.*
```

Comment:
Joissain tapauksissa .* importit on ok, (esim. ison testiluokan assertit), mutta java.util.* on kyllä ollut todella pitkään "ei näin", vaan importtaa kunkin oikeasti käytetin luokan erikseen.  (Tyylijuttuja).

Reply:
Korjasin tuon.